projects/canopy/src/styles/variables/_main.scss

```diff
@@ -21,9 +21,11 @@
   /* Links */
   --link-color: var(--color-super-blue);
   --link-hover-color: var(--color-super-blue-dark);
-  --link-visited-color: var(--color-super-blue-darkest);
-  --link-focus-background-color: var(--color-black);
```

Comment:
I've updated `--link-focus-background-color` to `--link-focus-bg-color`. I don't think it's a breaking change because it was only called in one of the mixins that has been changed.

Reply:
Just wondering about the scenario where an adopter of Canopy might be overriding `--link-focus-background-color` for themeing/syndication? Would we consider that to be a breaking change? If so, we could deprecate the old one and mark it for removal in our [upcoming deprecations and breaking changes](https://github.com/Legal-and-General/canopy/issues/196).

Reply:
You mean keep using `--link-focus-background-color` but mark it as deprecated? 
I just thought because this is already a breaking change it would be easier to do it all in one go instead of creating another breaking change later on.

Happy to do whatever you and @andy-polhill think it's best.

Reply:
Ah yes, that's a good point :-) let's keep it in this PR then - but I think we should add a note in the commit msg in the breaking change notes for this too, so it appears in the changelog/release notes. Maybe something like this?

```
Renamed --link-focus-background-color to --link-focus-bg-color
```

As @andy-polhill mentioned in #196, we could also remove the deprecated colours from #196 as part of this commit, effectively bundling up those changes in one breaking change as well 👉   https://github.com/Legal-and-General/canopy/issues/196#issuecomment-808138461